selectrum.el

```diff
@@ -83,18 +96,12 @@ INPUT is a string, CANDIDATES is a list of strings."
   #'selectrum-default-candidate-refine-function
   "Function used to decide which candidates should be displayed.
 Receives two arguments, the user input (a string) and the list of
-candidates (strings) as returned by
-`selectrum-preprocess-candidates-function'. Returns a new list of
+candidates (strings) as given by
+`selectrum--preprocessed-candidates'. Returns a new list of
```

Comment:
We can't have a public API depend on a private variable. It sounds in this case like the docstring can just be rewritten to not refer to implementation details.

Reply:
Good point, thanks!